examples/react-rescript/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@bun-examples/react-rescript",
+  "version": "0.0.1",
+  "dependencies": {
+    "@rescript/react": "^0.10.3",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "web-vitals": "^1.0.1"
+  },
+  "scripts": {
+    "build": "rescript",
```

Comment:
this is using `rescript`, which launches a Node.js process and not a Bun.js runtime process